<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biblioteca</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/new/prefix1/iconos/libro.png</normaloff>:/new/prefix1/iconos/libro.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="TituloTabla">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Transacciones</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="campo"/>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="az">
          <property name="text">
           <string>A-Z</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="za">
          <property name="text">
           <string>Z-A</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QStackedWidget" name="area">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableView" name="DatosTransaccion">
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableView" name="DatosClientes">
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QTableView" name="DatosLibros">
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QTableView" name="Datos">
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="tiempo">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="fecha">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hora">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="barra">
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>Tablas</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Transaccion</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Clientes</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Libros</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Autores</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTablas"/>
   <addaction name="separator"/>
   <addaction name="actionVer"/>
   <addaction name="separator"/>
   <addaction name="actionAgregar"/>
   <addaction name="actionEliminar"/>
   <addaction name="actionEditar"/>
   <addaction name="actionRecargar"/>
   <addaction name="separator"/>
   <addaction name="actionExcel"/>
   <addaction name="actionWord"/>
   <addaction name="actionPDF"/>
   <addaction name="separator"/>
   <addaction name="actionQuery"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionUsuario"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionTablas"/>
   </widget>
   <widget class="QMenu" name="menuAcciones">
    <property name="title">
     <string>Acciones</string>
    </property>
    <addaction name="actionAgregar"/>
    <addaction name="actionEliminar"/>
    <addaction name="actionEditar"/>
    <addaction name="actionRecargar"/>
    <addaction name="actionVer"/>
   </widget>
   <widget class="QMenu" name="menuCalculos">
    <property name="title">
     <string>Calculos</string>
    </property>
    <addaction name="actionQuery"/>
   </widget>
   <widget class="QMenu" name="menuExportar">
    <property name="title">
     <string>Exportar</string>
    </property>
    <addaction name="actionExcel"/>
    <addaction name="actionPDF"/>
    <addaction name="actionWord"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuExportar"/>
   <addaction name="menuVer"/>
   <addaction name="menuAcciones"/>
   <addaction name="menuCalculos"/>
  </widget>
  <action name="actionTablas">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/barra-lateral.png</normaloff>:/new/prefix1/iconos/barra-lateral.png</iconset>
   </property>
   <property name="text">
    <string>Tablas</string>
   </property>
  </action>
  <action name="actionAgregar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/mas.png</normaloff>:/new/prefix1/iconos/mas.png</iconset>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/eliminar.png</normaloff>:/new/prefix1/iconos/eliminar.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionVer">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/servidor.png</normaloff>:/new/prefix1/iconos/servidor.png</iconset>
   </property>
   <property name="text">
    <string>Ver</string>
   </property>
  </action>
  <action name="actionExcel">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/excel.png</normaloff>:/new/prefix1/iconos/excel.png</iconset>
   </property>
   <property name="text">
    <string>Excel</string>
   </property>
  </action>
  <action name="actionQuery">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/query-analysis.png</normaloff>:/new/prefix1/iconos/query-analysis.png</iconset>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
  </action>
  <action name="actionWord">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/word.png</normaloff>:/new/prefix1/iconos/word.png</iconset>
   </property>
   <property name="text">
    <string>Word</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/pdf.png</normaloff>:/new/prefix1/iconos/pdf.png</iconset>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/editar-informacion.png</normaloff>:/new/prefix1/iconos/editar-informacion.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </action>
  <action name="actionRecargar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/recargar.png</normaloff>:/new/prefix1/iconos/recargar.png</iconset>
   </property>
   <property name="text">
    <string>Recargar</string>
   </property>
  </action>
  <action name="actionUsuario">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/usuario.png</normaloff>:/new/prefix1/iconos/usuario.png</iconset>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
